src/styles/blocks/product.scss

```diff
@@ -0,0 +1,70 @@
+.product {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-items: center;
+    padding-inline: 3px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--light-text-color);
+    margin: 4px 0 7px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__priceText {
+    color: var(--light-text-color);
+  }
+
+  &__priceDollar {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__buy {
+    background-color: var(--blue-accent);
+    width: 100%;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. You've used padding. It's correct